stock_lot_scrap/tests/test_stock_lot_scrap.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+from odoo.tools.safe_eval import safe_eval
+
+
+@common.at_install(False)
```

Comment:
Instead of this line and the next one, use new syntax:

```
from odoo.tests import tagged


@tagged('post_install', '-at_install')
class TestStockLotScrap(...):
```